resources/analysisTools/snvPipeline/environments/tbi-lsf-cluster.sh

```diff
@@ -12,12 +12,15 @@ set -ue
 module load R/"${RSCRIPT_VERSION:?RSCRIPT_VERSION undefined}"
 module load python/"${PYTHON_VERSION:?PYTHON_VERSION undefined}"
 module load samtools/"${SAMTOOLS_VERSION:?SAMTOOLS_VERSION undefined}"
+module load bcftools/"${BCFTOOLS_VERSION:?BCFTOOLS_VERSION undefined}"
 module load htslib/"${HTSLIB_VERSION:?HTSLIB_VERSION undefined}"
 module load perl/"${PERL_VERSION:?PERL_VERSION undefined}"
 module load bedtools/"${BEDTOOLS_VERSION:?BEDTOOLS_VERSION undefined}"
 module load pypy/"${PYPY_VERSION:?PYPY_VERSION undefined}"
 
-source /odcf/cluster/virtualenvs/warsow/python_2.7.9_SNVCalling/bin/activate
+set +ue
+source /odcf/cluster/virtualenvs/paramasi/python_2.7.9_SNVCalling_pysam_0.15.2/bin/activate
```

Comment:
I understand that this is the easiest solution, but maybe a more portable, explicit, and safe solution is to add the specification of the virtual env as file (similar conda env export) into the repository.

... and did you actually consider using conda?

Reply:
Yes, I did try to port this to Conda when I initially started to work on this pipeline, but due to some bash issues, I faced problems running the workflow. So, I ended up copying the @GWarsow virtual environment. Maybe in a future version, this could be considered.